src/lib.rs

```diff
@@ -1,14 +1,186 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+use embedded_hal::blocking::i2c::{Write, WriteRead};
+
+const AK09915_ADDRESS: u8 = 0x0C;
+
+// Register Addresses
+#[allow(dead_code)]
```

Comment:
Why is this necessary ?

Reply:
just to supress unnused registers warning

Reply:
it's fine?